source/standard_retry_strategy.c

```diff
@@ -286,9 +286,14 @@ void s_standard_retry_strategy_on_retry_ready(struct aws_retry_token *token, int
         (void *)standard_retry_token,
         (void *)token);
     struct aws_retry_strategy *retry_strategy = token->retry_strategy;
+    aws_retry_token_acquire(standard_retry_token);
```

Comment:
Seems like maybe we don't need to increment ref count here?  (Looks like it's already guaranteed a ref count > 0 since we already have to do a release at the end of the function?)